alby.go

```diff
@@ -201,6 +201,9 @@ func (svc *AlbyOAuthService) CallbackHandler(c echo.Context) error {
 	svc.db.Save(&user)
 
 	sess, _ := session.Get("alby_nostr_wallet_connect", c)
+	if svc.cfg.CookieDomain != "" {
+		sess.Options.Domain = svc.cfg.CookieDomain
```

Comment:
can we derive this from the request somehow - so we don't need to set a config? 

e.g. if the request is on nwc.getalby.com we set the cookie on that host?

Reply:
Resolved [here](https://app.slack.com/client/T02SW936E95/threads/thread/C04FDDHNTJ6-1687764649.962909)